src/os/bluestore/BlueStore.cc

```diff
@@ -12999,37 +13013,41 @@ int BlueStore::_clone(TransContext *txc,
 		   newo->onode.is_pgmeta_omap() ? PREFIX_PGMETA_OMAP
 		   : PREFIX_OMAP,
 		   newo->onode.nid);
+    newo->onode.clear_omap_flag();
   }
   if (oldo->onode.has_omap()) {
+    const string& prefix =
+      newo->onode.is_pgmeta_omap() ? PREFIX_PGMETA_OMAP : PREFIX_OMAP;
     dout(20) << __func__ << " copying omap data" << dendl;
     if (!newo->onode.has_omap()) {
```

Comment:
This check isn't needed as we unconditionally clear omap for newo above

Reply:
fixed